### PR TITLE
Updating the EquilibriumBC

### PR DESCRIPTION
**Pull request by singhgp4321**
Ref: #52

### CODE REVIEW

**Comment by singhgp4321** [comment]
@lindsayad @cticenhour Could either of you please review this PR?

**Review by cticenhour — APPROVED** [review]
Thanks for making this change @singhgp4321. Tagging @humrickhouse.

File: doc/content/source/bcs/EquilibriumBC.md [hunk 1]
```diff
@@ -7,13 +7,13 @@
 This class strongly enforces the Dirichlet boundary condition
 
 \begin{equation}
-C_i = KP_i^p
+C_i = K_o \exp{\frac{-E_a}{RT}} P_i^p
```
[the remark this answers is not in the document]
Comment (singhgp4321) [reply doc/content/source/bcs/EquilibriumBC.md]: Added

File: include/bcs/EquilibriumBC.h [hunk 2]
```diff
@@ -20,21 +20,24 @@ class EquilibriumBC : public ADNodalBC
...
 
-  /// The equilibrium coefficient
-  const Real _K;
+  /// The solubility coefficient
+  const Real _Ko;
 
+  /// The solubility activation energy (J)
+  const Real _Ea;
+
+  /// The exponent in solubility law
```
[the remark this answers is not in the document]
Comment (singhgp4321) [reply include/bcs/EquilibriumBC.h]: Changed

File: include/bcs/EquilibriumBC.h [hunk 3]
```diff
@@ -20,21 +20,24 @@ class EquilibriumBC : public ADNodalBC
...
   /// The enclosure variable
   const ADVariableValue & _enclosure_var;
 
-  const Real _temp;
+  /// The temperature (K)
+  const ADVariableValue & _T;
 
-  /// Boltzmann's constant
-  const Real _kb;
+  /// Ideal gas constant (J/K-mol)
```
[the remark this answers is not in the document]
Comment (singhgp4321) [reply include/bcs/EquilibriumBC.h:38]: Thanks for the review!

**Review by cticenhour — APPROVED** [review]

**Pull request merged**